### DISCUSSION TITLE
Dependent dropdown fields.

### DISCUSSION THREAD

**Discussion by krish3888** [directus/directus#6375] [Feature Requests]
There should be an option for creating dependent dropdown field where one dropdown field is controlled by another dropdown field. 
For example, there are 2 dropdown fields.  First is Country (controlling field) and another is State (dependent field). Based on the selected country it will show us the state of that country only.

It will help and save our time. 
Thanks

**Comment by rijkvanzanten** [comment]
This is part of https://github.com/directus/directus/issues/3151 👍🏻
